### ISSUE TITLE
Terminal cursor doesn't display

### ISSUE THREAD

**Issue by maslade** [nealmckee/penumbra_vscode#1]
Hey there - I just checked out this theme. I really like the idea behind it and found the colors pleasing. I'm testing it in an outdoor environment with bright sunlight.

One thing that's tripping me up -- my terminal cursor doesn't appear. I'm not familiar enough with VS Code color settings or I'd override it myself. Any suggestions?

**Comment by nealmckee** [comment]
I’m glad you enjoy the color scheme! And also that it works well under difficult lighting despite being optimised for average conditions.

I already noticed this issue myself but sadly haven’t come around to fixing it due to some tight deadlines in my academic work.

There was some automation involved in creating the VSCode theme and there must have been a misaligned setting within that process as I set the cursor color correctly on my end and it got assigned the background color instead.

I know what to look for and just have to test which exact setting is the culprit to fix it. The problem will for sure be resolved with the next version (including some other small fixes). Expect it within the next month, by the end of the year at the latest (possibly sooner now that I know I’m not the only one using it).
